### PR TITLE
Add moveUp and moveDown functionality to ListViewItem component

### PR DESCRIPTION
**Pull request by hayes28**

### CODE REVIEW

**Comment by hayes28** [comment]
image
The 'order' number is not changing on the screen but does move if you enter something, just can't see it being edited. 
image
Two images to show the buttons work

**Comment by Oggal** [comment]
What do you mean by "The 'order' number is not changing on the screen but does move if you enter something, just can't see it being edited."?

File: tasm-staff-editor/src/components/ListViewComponents/ListViewComponents.js [hunk 1]
```diff
@@ -1,70 +1,147 @@
...
+        const curDocRef = doc(db, 'exhibits', exhibit.id) || null;
+        if (exhibit.order >= 0) {
+          const nextExhibit = sortedData[curIndex + 1];
+          if (nextExhibit) {
+            const nextDocRef = doc(db, 'exhibits', nextExhibit.id) || null;
+
+            const nextID = nextExhibit.exhibitID;
+            const curID = exhibit.exhibitID;
+            console.log("nextID", nextID);
+            console.log("curID", curID);
```
Comment (Oggal) [inline tasm-staff-editor/src/components/ListViewComponents/ListViewComponents.js]: Should we clean up these old console logs?

File: tasm-staff-editor/src/components/ListViewComponents/ListViewComponents.js [hunk 2]
```diff
@@ -1,70 +1,142 @@
...
-
   const [exhibitData, setExhibitData] = useState([]);
-  // let exhibitData = [];
+
+  const fetchData = async () => {
+    try {
+      const querySnapshot = await getDocs(collection(db, 'exhibits'));
+      let sortedData = querySnapshot.docs
+        .map(doc => ({ id: doc.id, order: doc.data().order, ...doc.data() }))
+        .sort((a, b) => a.order - b.order);
```
Comment (Oggal) [inline tasm-staff-editor/src/components/ListViewComponents/ListViewComponents.js:16]: minor thing, but should this use the sort function we have at line 122?

**Review by Oggal — APPROVED** [review]
I had one question, and I'll look at it a bit more before starting tomorrow if it's not merged in. But I would say it's ready!

Since I helped so much today I would love if anyone else from the team gave it a look over before we merge in.

**Pull request merged**